src/security/lockdown/lockdown.c

```diff
@@ -3,8 +3,42 @@
 #include <boot_device.h>
 #include <commonlib/region.h>
 #include <console/console.h>
+#include <drivers/efi/efivars.h>
 #include <bootstate.h>
 #include <fmap.h>
+#include <smmstore.h>
+
+#include <Uefi/UefiBaseType.h>
+
+static const EFI_GUID dasharo_system_features_guid = {
+	0xd15b327e, 0xff2d, 0x4fc1, { 0xab, 0xf6, 0xc1, 0x2b, 0xd0, 0x8c, 0x13, 0x59 }
+};
+
+/*
+ * Checks whether protecting vboot partition was enabled in BIOS.
+ */
+static bool is_vboot_locking_permitted(void)
+{
+#if CONFIG(DRIVERS_EFI_VARIABLE_STORE)
+	struct region_device rdev;
+	enum cb_err ret;
+	uint8_t var;
+	uint32_t size;
+
+	if (smmstore_lookup_region(&rdev))
+		return false;
+
+	size = sizeof(var);
+	ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "LockBios", &var, &size);
+	if (ret != CB_SUCCESS)
+		return true;
```

Comment:
What should be the default here? The default value of `BIOS Lock` in the setup menu is disabled, and it seems that the variable is created only after first toggling it. We enable bios lock here if it doesn't exist, so tianocore displays a wrong value by default.

Reply:
We should probably initialize it correctly in TianoCore. @SergiiDmytruk WDYT?

Reply:
Right. Always initializing is better than relying on defaults matching each other.

Reply:
Related PR: https://github.com/Dasharo/DasharoModulePkg/pull/3